coffeebuddy/__init__.py

```diff
@@ -27,7 +32,9 @@ def create_app(config=None):
     if app.config['ENV'] in ('development', 'prefilled') or app.testing:
         app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///:memory:'
     else:
-        app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///coffee.db'
```

Comment:
Can you please add a configuration option to have the sqlite database still available. E.g.
```python
elif app.config['DB_BACKEND'] == 'sqlite':
    app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///coffee.db'
else:
        db_user = socket.gethostname()
```